### DISCUSSION TITLE
How to define a base avatar and default avatar ?

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3075] [Hubs Cloud Questions] [answered]
Hello 

All is in the question . I have a aws hubs cloud and when user are connected for the first time they have little duck avatar . How to change that in the admin panel ? 

Thx

**Comment by robinkwilson** [comment, marked as answer]
1. Go to Admin Panel > Approved Avatars 
1. Click "Edit" on the avatar you'd like to set as default
1. Add a tag labelled `default` (see screenshot)

Alternatively _while you are importing_ the avatars, there is a checkbox to select a few avatars as default.

![image](https://user-images.githubusercontent.com/29560735/94943557-beebf780-048c-11eb-8620-7bdbef0c7281.png)
